static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -22,73 +18,63 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   // Only "One other issue appears in the same trace. View Full Trace (X issues)" would show up
   return (
     <Fragment>
-      {isLoadingGroupData ? (
-        <LoadingIndicator mini />
-      ) : (
-        groupData && (
-          // XXX: Determine plan for analytics
-          <TraceIssueLinkContainer
-            to={{
-              pathname: `/organizations/${organization.slug}/issues/${issueId}/events/${event.id}/`,
-              query: {
-                referrer: 'issues_trace_issue',
-              },
-            }}
-          >
-            <TraceIssueProjectBadge>
-              {project ? (
-                <ProjectBadge
-                  project={project}
-                  avatarSize={avatarSize}
-                  hideName
-                  disableLink
-                />
-              ) : (
-                <Placeholder
-                  shape="rect"
-                  width={`${projectBadgeSize}px`}
-                  height={`${projectBadgeSize}px`}
-                />
-              )}
-            </TraceIssueProjectBadge>
-            <TraceIssueDetailsContainer>
-              <NoOverflowDiv>
-                <TraceIssueEventTitle>
-                  {groupData.metadata.title || groupData.metadata.type}
-                </TraceIssueEventTitle>
-                <TraceIssueEventTransaction>
-                  {event.transaction}
-                </TraceIssueEventTransaction>
-              </NoOverflowDiv>
-              <NoOverflowDiv>{groupData.metadata.value}</NoOverflowDiv>
-            </TraceIssueDetailsContainer>
-          </TraceIssueLinkContainer>
-        )
-      )}
+      {/* // XXX: Determine plan for analytics */}
+      <TraceIssueLinkContainer
+        to={{
+          pathname: `/organizations/${organization.slug}/issues/${issueId}/events/${event.id}/`,
+          query: {
+            referrer: 'issues_trace_issue',
+          },
+        }}
+      >
+        <TraceIssueProjectBadge>
+          {project ? (
+            <ProjectBadge
+              project={project}
+              avatarSize={avatarSize}
+              hideName
+              disableLink
+            />
+          ) : (
+            <Placeholder
+              shape="rect"
+              width={`${projectBadgeSize}px`}
+              height={`${projectBadgeSize}px`}
+            />
+          )}
+        </TraceIssueProjectBadge>
+        <TraceIssueDetailsContainer>
+          <NoOverflowDiv>
+            <TraceIssueEventTitle>{title}</TraceIssueEventTitle>
+            <TraceIssueEventTransaction>{event.transaction}</TraceIssueEventTransaction>
+          </NoOverflowDiv>
+          <NoOverflowDiv>{subtitle}</NoOverflowDiv>
+        </TraceIssueDetailsContainer>
+      </TraceIssueLinkContainer>
     </Fragment>
   );
 }
 
+function getTitleSubtitle(event: TimelineEvent) {
+  let title;
+  let subtitle;
+  if (event['event.type'] === 'error') {
+    title = event.title.split(':')[0];
```

Comment:
I'm okay with this as a temporary hack, but if we go forward with this I'd want to make sure that this matches the logic we have in the backend

Reply:
It's on my list of tasks.